routes/web.php

```diff
@@ -35,14 +35,65 @@
     return view('pub.more');
 })->name('pub.more');
 
+
 Route::group(['prefix' => 'admin'], function(){
 
     Route::get('dashboard', function() {
         return view('admin.index');
     })->name('admin.index');
 
+/*
+	Route::get('users', function() {
+        return view('admin.users');
+    })->name('admin.users'); */
+
 });
 
+
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+
+Route::group(['prefix' => 'admin', 'namespace' => 'Admin'], function(){
+    Route::get('/', 'AdminController@index');
+    Route::get('/login', 'AdminLoginController@showLoginForm');
+    Route::post('/login', 'AdminLoginController@login');
+    Route::get('/logout', 'AdminLoginController@logout');
+
+	Route::get('users', 'UserController@index');
+	Route::get('posts', 'PostController@index');
+
+
+
+
+
+
+
+
+/*
+ Route::get('posts', function () {
+        return view('posts', [
+            'posts' => Post::orderBy('created_at', 'asc')->get()
+        ]);
+    });
+
+*/
+
+	
+	Route::get('create', 'PostController@createIndex');
+	Route::get('posts/{id}/edit', 'PostController@editIndex');
+
+	Route::get('posts', 'PostController@index');
```

Comment:
why is this under the admin group?

Reply:
Used in /admin/posts. Lists current posts with edit and delete options.

Reply:
WHy can't we just list the edit and delete options when we render them for all users? It's just that admins can delete/edit all posts